WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -116,13 +117,11 @@
 
 ## Bravery settings
 
-- [ ] Check that HTTPS Everywhere works by loading `http://https-everywhere.badssl.com/`
-- [ ] Turning HTTPS Everywhere off or shields off both disable the redirect to `https://https-everywhere.badssl.com/`
 - [ ] Check that block ad and unblock ad works on `http://slashdot.org`
 - [ ] Check that toggling to blocking and allow ads works as expected
 - [ ] Verify that tapping through a cert error in `https://badssl.com` works
-- [ ] Verify that Safe Browsing works `https://www.raisegame.com`
-- [ ] Enable `Block Scripts` globally from `Settings`.  Visit `https://twitter.com`, nothing should load. Tap on `Shields` and disable `Block Scripts`; the page should load properly.
+- [ ] Verify that Safe Browsing works with `https://testsafebrowsing.appspot.com/`
```

Comment:
Not sure we can use this here but this used to be for Chromium safe browsing tests. Same is applicable for WkWebView